.github/workflows/release-please.yml

```diff
@@ -0,0 +1,49 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
So, merging to main will always trigger the release, is that right?

Reply:
Ah apologies, I should have linked here the actual process definition, you can see it in https://github.com/grafana/deployment_tools/pull/177384, this is the part specifically for the app [here](https://github.com/grafana/deployment_tools/blob/sm-release-process-docs/docs/synthetic-monitoring/release-process.md#synthetic-monitoring-app).